new_course/1_programming_fundementals/input_and_output.md

```diff
@@ -1,23 +1,73 @@
 <!-- This lesson will cover how to output things to the screen in Ruby and how to get input from the user. -->
 
 ### Introduction
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+When we execute a computer program in Ruby Programming Language, most of the times it is required to either **output** or **input** a piece of information to a computer screen.
```

Comment:
I don't think we need quite as much detail in this introduction. What about changing this to something like:

"To create programs that are user friendly and interactive you'll need to know how to output data to a screen and how to get input from a user. We'll cover all the most common ways to achieve this in Ruby in this Lesson."